benchmarks/CMakeLists.txt

```diff
@@ -50,6 +50,10 @@ macro(fiber_add_mpi_test)
     unset(_fiber_COMMAND)
 endmacro()
 
+if (FIBER_ENABLE_FFTE)
+    link_directories(${CMAKE_Fortran_IMPLICIT_LINK_DIRECTORIES})
```

Comment:
Just one word about this implicit variable.
What is it for?

Reply:
For Gfortran, `implicit` library is `-lgfortran`. For IBM, it's `-lxlf` and so on. And they are usually hidden in strange place so I need `LINK_DIRECTORIES` added to tell linker where to find them.

Reply:
Forgot the links: https://cmake.org/cmake/help/v3.10/variable/CMAKE_LANG_IMPLICIT_LINK_DIRECTORIES.html https://cmake.org/cmake/help/v3.10/variable/CMAKE_LANG_IMPLICIT_LINK_LIBRARIES.html

Reply:
Thank you for the explanation.